tasks/sast-go.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gosec-check
+spec:
+  workspaces:
+    - name: workspace
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
+  steps:
+    - name: gosec-check
+      image: quay.io/redhat-appstudio/hacbs-test:stable
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      script: |
+        /usr/local/go/bin/gosec -no-fail -fmt=sarif -out=gosec_output.json $(workspaces.workspace.path)/... | tee gosec_output.txt
```

Comment:
that `/...` (tree dots, `cd ...` returns `cd: ...: No such file or directory`) is handled by gosec or is it bug?

Reply:
Yeah I was puzzled by that too, but the [README](https://github.com/securego/gosec#usage) for gosec says this:
>  To recursively scan from the current directory you can supply ./... as the input argument.